erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -52,6 +52,13 @@ def on_submit(self):
 		self.update_loan()
 		self.update_inter_company_jv()
 
+	def before_print(self):
+		gl_entries = frappe.get_list("GL Entry",filters={"voucher_type": "Journal Entry",
```

Comment:
why create another variable?
```suggestion
		self.gl = frappe.get_list("GL Entry",filters={"voucher_type": "Journal Entry",
```

Reply:
Also, maybe name it `self.gl_entries`?